services/observability/CHANGELOG.md

```diff
@@ -0,0 +1,123 @@
+## v0.12.0 (2024-08-21)
+- **Breaking change**: STACKIT Argus service has been renamed to STACKIT Observability
```

Comment:
Also I think this not a breaking change, it's a release of a new module (the old `argus` module will still be usable with all its released versions/tags)

Reply:
In the Release Notes of the next GitHub release we could add something like a

> [!Warning]
> The STACKIT Argus service was renamed to Observability. There is a new `observability` service in the SDK which offers the same functionality as the deprecated `argus` service.
> SDK updates from now on will be released on the new `observability` service, meaning `argus` will no longer get updates.
> Please migrate to ...

Reply:
I changed the changelog and also added the highlight about renaming to the main changelog